scripts/cc-cli/src/utils/account.ts

```diff
@@ -1,21 +1,56 @@
+import { mnemonicValidate } from "@polkadot/util-crypto";
 import { ApiPromise, Keyring } from "creditcoin-js";
-import { OptionValues } from "commander";
-import { readFileSync } from "fs";
+import prompts from "prompts";
 
 export function initKeyringPair(seed: string) {
   const keyring = new Keyring({ type: "sr25519" });
   const pair = keyring.addFromUri(`${seed}`);
   return pair;
 }
 
-export function getSeedFromOptions(options: OptionValues) {
-  if (options.seed) {
-    return options.seed;
-  } else if (options.file) {
-    return readFileSync(options.file).toString();
-  } else {
-    throw new Error("Must specify either mnemonic phrase or file as an option");
+export async function getStashSeedFromEnvOrPrompt() {
+  return await getSeedFromEnvOrPrompt(
+    process.env.CC_STASH_SEED,
+    "Specify a seed phrase for the Stash account"
+  );
+}
+export async function getControllerSeedFromEnvOrPrompt() {
+  return await getSeedFromEnvOrPrompt(
+    process.env.CC_CONTROLLER_SEED,
+    "Specify a seed phrase for the Controller account"
+  );
+}
+
+export async function getSeedFromEnvOrPrompt(
+  envVar?: string | undefined,
+  promptStr?: string | null
+) {
+  if (envVar) {
+    if (mnemonicValidate(envVar)) {
+      return envVar;
+    } else {
+      console.log(
+        "Error: Seed phrase provided in environment variable is invalid."
+      );
+      process.exit(1);
+    }
+  }
+  const seedPromptResult = await prompts([
+    {
+      type: "invisible",
+      name: "seed",
+      message: promptStr ? promptStr : "Enter seed phrase",
+      validate: (seed) => mnemonicValidate(seed),
```

Comment:
Not sure if that's an issue but is definitely confusing for me:

1) Issue the `bond` command, no ENV defined
2) Type `1234` for seed, press Enter
3) The error message `Please Enter A Valid Value` appears - that's expected. 
4) Green cursor appears on the line of the error message. I pressed Enter
5) Green cursor now appears back on the line `Specify a seed phrase for the Stash acount`
6) I entered my seed phrase, to the best of my abilities
7) Error message is still there. IDK if I made a mistake again or if the command got stuck

```
$ node dist/index.js bond -u ws://127.0.0.1:9946   -c 5EEoJZYPpTYGX1t3shaQbqhRmqvdvoaTycY1gJ4TB8spDgPp -a 1000005 -r Staked
2023-06-21 15:08:13        API/INIT: RPC methods not decorated: creditcoin_hashrate, task_getOffchainNonceKey
2023-06-21 15:08:13        API/INIT: creditcoin-node/222: Not decorating unknown runtime apis: TaskApi/1
? Specify a seed phrase for the Stash account › 
› Please Enter A Valid Value
```

Reply:
I spent way too long thinking it was some async bug.